modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -77,7 +77,7 @@ def get_filtered_unique_errors
 
       unique_errors.each do |ue|
         filtered_error_ids << ue[:id] unless NO_INVESTIGATION_ERROR_TEXT.any? do |text|
-          ue[:evss_response]&.include?(text)
+          ue[:evss_response].to_s&.include?(text)
         end
```

Comment:
In the past some of these errors have gotten us because they can come in many forms, casting it to a string felt safer for handling it error free.